scss/modules/_feedback.scss

```diff
@@ -0,0 +1,50 @@
+// Feedback module
+
+.feedback {
+  @include transition(bottom .5s);
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>`.5` should be written with a leading zero as `0.5`